client/state/reader/streams/test/reducer.js

```diff
@@ -26,7 +26,7 @@ import {
 	PENDING_ITEMS_DEFAULT,
 } from '../reducer';
 
-jest.mock( 'calypso/lib/warn', () => () => {} );
+jest.mock( '@automattic/warn', () => () => {} );
```

Comment:
I'm wondering why we do all this test mocking. Would the tests fail if we didn't mock?

Reply:
This is actually a recommended practice in the README:

````
If you'd like to hide output within a test,
add this line to the top of your test file:

```javascript
jest.mock( '@automattic/warn', () => () => {} );
```
````